packages/shared/src/components/post/write/SquadsDropdown.tsx

```diff
@@ -15,7 +17,12 @@ export function SquadsDropdown({
   selected,
 }: SquadsDropdownProps): ReactElement {
   const { squads } = useAuthContext();
-  const squadsList = squads?.map(({ name }) => name) ?? [];
+  const squadsList = squads?.reduce((acc, squad) => {
```

Comment:
Would using `.filter` be a more suitable option?

Reply:
Hmm, since we cast to [] I thought reduce might make most sense.
But I guess filter would also work to be honest.